### ISSUE TITLE
Functions tagged with @Preview shouldn't be flagged by ModifierMissing rule

### ISSUE THREAD

**Issue by manuel-martos** [twitter/compose-rules#43]
**Describe the bug**
Code like this gets flagged by ModifierMissing rule:

```
@Preview
@Composable
fun PreviewSomething() {
    LazyColumn {
        // ... whatever
    }
}

```

Wouldn't it be nice to don't have this false issue flagged by this rule?

**Comment by mrmans0n** [comment]
This was fixed in #42, and this issue is a clone of #41 who flagged it first 😄
